spotless/license

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Mask-Android
+ *
+ *  Copyright (C) DimensionDev and Contributors
+ * 
+ *  This file is part of Mask-Android.
+ * 
+ *  Mask-Android is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
```

Comment:
Mask Android is under AGPLv3 not GPLv3
Reference file header template: https://github.com/licenses/license-templates/blob/master/templates/agpl3-header.txt